src/components/MessageInput/MessageInputFlat.js

```diff
@@ -196,14 +197,18 @@ class MessageInputFlat extends PureComponent {
                     height="28"
                     xmlns="http://www.w3.org/2000/svg"
                   >
+                    <title>{t('Open emoji picker')}</title>
```

Comment:
making sure these new texts are included in the i18n translation files

Reply:
They are, they just weren't used in all the MessageInput components (this makes it more consistent)